FIT-15

qrillich opened the pull request:
RoleAPI

Commit:
RoleAPI
Добавлено:
RoleController.java - NonNull
RoleDTO.java
IRoleRepository.java
IRoleService.java
RoleService.java
IRoleMapper.java

Необходимо доделать:
-Unit тесты.

ws-eliseev commented on src/main/java/ws/eliseev/fitness/controller/RoleController.java:
```diff
@@ -0,0 +1,165 @@
+            @ApiResponse(responseCode = "404", description = "Роль не найдена")})
+    public ResponseEntity<RoleDTO> getRoleById(@PathVariable("id") Long id) {
+        final Optional<Role> gotRole = roleService.findRoleById(id);
+        log.info("Get role by id");
```
log.info("Get role by id: {}, role: [{}]", () -> id, () -> Optional.ofNullable(gotRole));

Commit:
Merge branch 'devel' into FIT-15

Commit:
Merge branch 'devel' into FIT-15

Commit:
RoleAPI
Изменено:
RoleController.java - mapper убран в сервис
IRoleRepository.java - JavaDOC
IRoleService.java - mapper перенесен из контроллера
RoleService.java - */*

Необходимо доделать:
-Unit тесты.

Commit:
RoleAPI
Изменено:
RoleController.java

Необходимо доделать:
-Unit тесты.
